### ISSUE TITLE
Add Visible Keyboard Focus

### ISSUE THREAD

**Issue by hannahackermans** [smbjorklund/cellproject#63]
### Current State
There is no visible indication of keyboard focus for most elements, except the search button which has the typical blue highlight when it receives focus. The green headings turn grey when receiving focus but this is barely visible. 

### Proposed changes to meet the success criteria
Add keyboard focus: when you focus on a user interface component, the control should be highlighted. I suggest using the typical default blue highlight, since this is clear against the white background.
